app.rb

```diff
@@ -1,197 +1,116 @@
+require 'json'
+require_relative './models/person'
 require_relative './models/student'
-require_relative './models/teacher'
-require_relative './models/book'
 require_relative './models/rental'
+require_relative './models/book'
+require_relative './models/teacher'
+require_relative './models/classroom'
+require_relative './models/data'
 
 class App
+  include Presist
   def initialize
-    # print commands
-    @cmd = {
-      '1': 'List all books',
-      '2': 'List all people',
-      '3': 'Create a person',
-      '4': 'Creat a book',
-      '5': 'Create a rental',
-      '6': 'List all rentals for a given person id',
-      '7': 'Exit'
-    }
-
-    # list of books
-    @books = []
-
-    # list of people
-    @people = []
+    @books = load_books
+    @people = load_people
+    @rentals = load_rentals
   end
 
-  # display commands
-  def display_cmd
-    @cmd.each do |index, command|
-      puts "#{index} - #{command}"
+  def handle_action(option)
+    case option
+    when '1'
+      list_books
+    when '2'
+      list_people
+    when '3'
+      create_person
+    when '4'
+      create_book
+    when '5'
+      create_rental
+    when '6'
+      list_rentals
+    else
+      puts 'That is not a valid option'
     end
   end
 
-  # print out all books
-  def list_all_books
-    if @books.empty?
-      puts 'Please insert books first!'
-    else
-      @books.each_with_index do |book, index|
-        puts "#{index}) Title: #{book.title}, Author: #{book.author}}"
-      end
-    end
+  def list_books
+    @books.each { |book| puts book }
   end
```

Comment:
- [ ] The issue highlighted by the previous reviewer is still there. When I select option 1 to list the books it just displays the same message in the image from the previous reviewer. You can fix that by change this code to this.

  ```
  def list_books
    @books.each { |book| puts "#{book.title} #{book.author}" }
  end
  ```